client/components/dashboard/dashboard-directive.html

```diff
@@ -32,6 +32,11 @@
             <span class="glyphicon glyphicon-refresh pk-widget-button"
                   ng-show="widget.model.type === widgetFactorySvc.widgetTypes.CHART"
                   ng-click="clickRefreshWidget($event, widget)"></span>
+            <span class="fa fa-camera pk-widget-button"
+                  ng-show="widgetSvc.isCopyableAsImage(widget)"
+                  ng-click="clickCopyAsImage($event, widget)">
+              <md-tooltip>Copy the current chart as a PNG, which you can paste into your favorite program.</md-tooltip>
```

Comment:
How about just "Copy chart image to clipboard"? The current one is a bit confusing, the PNG itself isn't really visible anywhere and seems like an implementation detail.